srcpkgs/ergo/oragono.INSTALL

```diff
@@ -0,0 +1,10 @@
+case "$ACTION" in
+post)
+	if [ "$UPDATE" = "yes" ]; then
+		[ -f etc/oragono.conf ] && mv etc/oragono.conf etc/ergo.conf
+		if [ -d var/lib/oragono ]; then
+			mv -T var/lib/oragono var/lib/ergo
+			chown -R _ergo:_ergo var/lib/ergo
+		fi
+	fi ;;
+esac
```

Comment:
Strike this file. Automated moving of prior configuration is a nightmare; let users manually migrate.